src/metabase/util/log.clj

```diff
@@ -78,17 +86,23 @@
   You shouldn't have to use this directly; prefer the level-specific macros like [[info]]."
   {:arglists '([level message & more] [level throwable message & more])}
   [level x & more]
-  (macros/case
-    :cljs (glogi-logp (str *ns*) level x more)
-    :clj  (tools-logp *ns*       level x more)))
+  `(do
```

Comment:
I don't think you need the do here, you could just have this top level since the body of a defn has an implicit do

Reply:
This is a `defn` that's used for macroexpansion output